src/app/layout.tsx

```diff
@@ -68,7 +71,7 @@ export default async function RootLayout({
         <SkipLink href="#main" />
         {children}
         <GovUkInitAll />
-        {process.env.GTM && process.env.GA && (
+        {process.env.GTM && process.env.GA && acceptAnalytics && (
```

Comment:
Would you also mind splitting the `<GoogleTagManager />` and `<GoogleAnalytics />` apart so we don't need to have both env vars set and can be more flexible? 🙏 

We can load GA through GTM and make it easier to manage in one place! (I've already made that change in GTM)